kansha/services/assetsmanager.py

```diff
@@ -8,13 +8,23 @@
 # this distribution.
 #--
 
+from nagare import presentation
+from paste import fileapp
+from webob.exc import WSGIHTTPException
```

Comment:
Sort the imports in this order, each category separated by an empty line:

1. the Python standard library modules
2. the Pypi modules ('nagare' included)
3. the modules of this project
4. the modules of this component

And, in each category, sort for the shorter to the longer line